public/js/toc.jsx

```diff
@@ -0,0 +1,18 @@
+/* global React */
+
+import TableOfContents from './components/table-of-contents';
+
+export default function init(maxDepth) {
+    var tocNode = document.getElementById('toc');
```

Comment:
Any way to preprocess the markup instead of passing in the mount node by id?  I don't know react well, just seems likely tight coupling with that ID.

Probably not a big deal giving the scope of this.. and we were doing it previously the same way so not a blocker.